snet_cli/arguments.py

```diff
@@ -550,26 +565,34 @@ def add_mpe_client_options(parser):
     subparsers = parser.add_subparsers(title="Commands", metavar="COMMAND")
     subparsers.required = True
 
-    def add_p_deal_with_response(p):
+    def add_p_set1_for_call(p):
```

Comment:
Can we make the name more descriptive. Not sure what set1 means in this context.

Reply:
It is intentional. This function represent common parameters for ```call``` and ```call-lowlevel```. But these parameters too diverse to find a common name for them, and I don't want anybody to reuse this function, so I called it "set1"...